components/ChipTextField/examples/MDCChipTextField.m

```diff
@@ -13,14 +13,22 @@
 // limitations under the License.
 
 #import "MDCChipTextField.h"
+
 #import "MaterialChips.h"
+#import "MDCChipTextFieldScrollView.h"
+
+//static CGFloat const kChipsSpacing = 10.0f;
 
-@interface MDCChipTextField ()
+@interface MDCChipTextField () <MDCChipTextFieldScrollViewDataSource>
 
-@property(nonatomic, strong) UIView *chipsView;
-@property(nonatomic) CGFloat insetX;
-@property(nonatomic, strong) NSLayoutConstraint *leadingConstraint;
-@property(nonatomic, strong) NSMutableArray<MDCChipView *> *chips;
+@property (nonatomic, weak) MDCChipTextFieldScrollView *chipsContainerView;
+@property (nonatomic) CGFloat chipsContainerHorizontalOffsetForTextEditing;
+@property (nonatomic) CGFloat chipsContainerHorizontalWidthForTextEditing;
+
+@property (nonatomic, readwrite, weak) NSLayoutConstraint *chipContainerViewConstraintLeading;
```

Comment:
I think this should be strong not weak.

Reply:
Discussed offline. Will keep the properties for constraints to be `weak`.

Reply:
For posterity (and those of us in the audience), could you briefly explain why these should be `weak`?